src/App.vue

```diff
@@ -25,10 +28,20 @@ function isLeftPanelRetractable() {
 
 <template>
   <main class="h-screen flex">
-    <aside class="z-10 absolute">
-      <SidePanel :is-retractable="isLeftPanelRetractable()">
+    <aside
+      class="z-10 absolute"
+      :class="panelStore.typePanelDisplay == 'right' ? 'right-0' : 'left-0'"
+    >
+      <SidePanelLeft
```

Comment:
Actually, I am thinking to use SidePanel for both Left Panel and Right Panel. This way, we can have one RouterView only. If we separated like this, both panels will have the same content, and probably will do more stuff than needed.

Reply:
I think if we have one panel, we can change the side of map according to the value of panelStore.typePanelDisplay
For now, i have merge right and left panel in the same 
With the futur tickets we can see if it's necessary to separate the two panel,  and use the ticket : https://jira.camptocamp.com/browse/GSREN3D-245 to update the right panel